ui/app/common/patient/filters/age.js

```diff
@@ -4,11 +4,11 @@ angular.module('bahmni.common.patient')
 .filter('age', function () {
     return function (age) {
         if (age.years) {
-            return age.years + " y";
+            return age.years + " year" + (age.years > 1 ? "s" : "");
```

Comment:
this would not be i18n. not that it as before .. but now its year, month, day - any specific reason?

Reply:
also why complicate with plural (s) ..

Reply:
Yes.. agree..should use an i18n template string: `%age_in_years (%sex)`

Reply:
@angshu @gsluthra Added the translations